### ISSUE TITLE
sanddance in React breaking UI

### ISSUE THREAD

**Issue by anilSimon123** [microsoft/SandDance#686]
Hi,

I have created a react application using create-react-app.

I am taking a bit of a different approach because i want some space to manipulate the data first.
This is my App.js

```
import React, { useEffect } from 'react';
import ReactDOM from 'react-dom';
import * as SandDanceReact from '@msrvida/sanddance-react';
import * as vega from 'vega';
import * as fluentui from '@fluentui/react';
import { Explorer, use } from '@msrvida/sanddance-explorer';
import "./style.css";

SandDanceReact.use(React, ReactDOM, vega);

fluentui.initializeIcons();

use(fluentui, React, ReactDOM, vega);

const data = [
  { a: 1, b: "c1" },
  { a: 1, b: "c2" },
  { a: 2, b: "c3" },
  { a: 3, b: "c4" }
];
const explorerProps = {
  logoClickUrl: 'https://microsoft.github.io/SandDance/',
  mounted: explorer => {
    explorer.load(data);
  }
};
function App() {
  ReactDOM.render(React.createElement(Explorer, explorerProps), document.getElementById('explorer'));

  return (
    <div>
        <div width= "100vw" height="100vh" id="innerContainer">This content will be replaced.</div>
    </div>
  );
}

export default App;
```

package.json

```
{
  "name": "react-sanddance",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "react": "17",
    "react-dom": "17",
    "react-scripts": "5.0.1",
    "vega": "^5.25.0",
    "web-vitals": "^2.1.4",
    "@fluentui/react": "^8",
    "@msrvida/sanddance-explorer": "^4.1.6",
    "@msrvida/sanddance-react": "^4.0.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```
another attempt at App.js
```

import React, { useEffect } from 'react';
import ReactDOM from 'react-dom';
import * as SandDanceReact from '@msrvida/sanddance-react';
import * as vega from 'vega';
import * as fluentui from '@fluentui/react';
import { Explorer, use } from '@msrvida/sanddance-explorer';
import "./style.css";



function App() {
  SandDanceReact.use(React, ReactDOM, vega);
  fluentui.initializeIcons();
  use(fluentui, React, ReactDOM, vega);
  const data = [
    { a: 1, b: "c1" },
    { a: 1, b: "c2" },
    { a: 2, b: "c3" },
    { a: 3, b: "c4" }
  ];

  const explorerProps = {
    logoClickUrl: 'https://microsoft.github.io/SandDance/',
    mounted: explorer => {
      explorer.load(data);
    }
  };

  return (
    <div>
      <h1>Main App Component</h1>
      <div id="innerContainer">
        <Explorer {...explorerProps} />
      </div>
    </div>
  );
}

export default App;

```

style.css
```
.sanddance-explorer {
  height: 800px; /*whatever height you need */
}

```
Referring this.
https://microsoft.github.io/SandDance/docs/sanddance-explorer/v4/

![image](https://github.com/microsoft/SandDance/assets/19643770/b5e095a0-a1c6-4de1-9ec9-e02c55164ad8)

Am I missing any dependency? Is what I am doing even possible?
Are there any working examples I can download and test it and play around that anyone is aware of?  
```[tasklist]
### Tasks
```

**Comment by danmarshall** [comment]
Hi @anilSimon123 , can you provide a GitHub repo?

**Comment by danmarshall** [comment]
Update: I created a repo with create-react-app here: https://github.com/danmarshall/create-react-sanddance

You'll need to add the stylesheet from the distribution:
```js
import "@msrvida/sanddance-explorer/dist/css/sanddance-explorer.css"
```
see #305 

I will add this to the readme, thanks for the report.

**Comment by danmarshall** [comment]
Fixed in #687
